.github/workflows/styles.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install specific toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2023-04-10
+          toolchain: nightly-2024-01-19
```

Comment:
applies to all of them of course.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1305/commits/969a411a23b1cea27856a14cd0a064efb70d1321